### PR TITLE
Frontend/Backend Deployment

### PR DESCRIPTION
**Pull request by billxsheng**
**PR Contents**
- Added Netlify config for frontend deployment
- Added Heroku config for backend deployment
- Changed backend URL in Lobby.js
- Added CORS dependency to backend to avoid production errors

**Areas of Inspection**
- Suggestions for configuring local/production endpoints?

### CODE REVIEW

**Review by varoongupta — APPROVED** [review]
looks nice!

File: wewatch-client/src/components/Lobby/Lobby.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { Link } from "react-router-dom";
 
 import './Lobby.css';
 
-const ENDPOINT = 'http://localhost:5000';
+// change to http://localhost:5000 for local development
+const ENDPOINT = 'https://wewatch-server.herokuapp.com/';
```
Comment (dlhxu) [inline wewatch-client/src/components/Lobby/Lobby.js:9]: We can change this to be set by a config file in future sprints, looks good for now!

**Review by dlhxu — APPROVED** [review]
Looks good to me!

**Review by jeffreyquiambao — APPROVED** [review]
Can't wait to demo this to the class!

**Review by rosakng — APPROVED** [review]
i love netlify!

**Pull request merged**